src/Compilers/CSharp/Portable/Symbols/TupleFieldSymbol.cs

```diff
@@ -58,17 +58,17 @@ internal static FieldSymbol GetUnderlyingField(int containingTupleArity, NamedTy
                 containingLinkSize = tupleRemainder;
             }
 
+            NamedTypeSymbol linkType = TupleTypeSymbol.GetNestedTupleType(containingUnderlyingType, fieldChainLength - 1);
+
             // PROTOTYPE(tuples) Constructing a TupleTypeSymbol with TupleFieldSymbols should not produce diagnostics for missing well-known members and type members.
             //                      The error handling should be shifted to lowering. This will help when we implement metadata loading.
             WellKnownMember wellKnownTypeMember = TupleTypeSymbol.GetTupleTypeMember(containingLinkSize, fieldRemainder);
-            var linkField = (FieldSymbol)Binder.GetWellKnownTypeMember(binder.Compilation, wellKnownTypeMember, diagnostics, syntax: syntax);
+            var linkField = (FieldSymbol)Binder.GetWellKnownTypeMember(binder.Compilation, wellKnownTypeMember, diagnostics, type: linkType.OriginalDefinition, syntax: syntax);
```

Comment:
I don't understand why this method shouldn't report diagnostics. 
We need to report missing members diagnostics during lowering and also need to call this method to get the members. It seems natural to do the two together.